bin/publish.ls

```diff
@@ -0,0 +1,138 @@
+require! {
+  'chalk' : {bold, red, cyan, yellow, green}
+  'inquirer'
+  'jsonfile'
+  'semver'
+  'shelljs/global'
+}
+
+{name, version} = jsonfile.readFileSync './exocom-server/package.json' # not sure about this, using exocom-server b/c there is no root package.json
+
+if process.argv.length != 3
+  display-help!
+
+level = process.argv[2]
+display-help! if level in ['-h, --help']
+
+target-version = semver.inc version, level
+unless target-version
+  console.log "\n#{bold red 'Error:'} #{bold cyan level} #{red 'is not a valid version increment'}"
+  display-help!
+
+console.log "\nYou are about to bump #{green bold name} version #{cyan bold version} up to #{cyan bold target-version}\n"
+
+question =
+  type: 'list'
+  name: 'continue'
+  message: 'Are you sure?'
+  choices: ['yes', 'no']
+inquirer.prompt([question]).then (answer) ->
+  if answer.continue == 'no'
+    console.log '\nAborting ...\n'
+    process.exit!
+
+  console.log!
+
+  # Ensure no open files
+  open-files = exec "git status --porcelain", {silent: true}
+  if open-files then console.log red 'Please commit all files before releasing' ; process.exit 1
+
+  # Ensure on master
+  current-branch = exec "git rev-parse --abbrev-ref HEAD", {silent: true}
+  if current-branch.trim! isnt 'master' then console.log red 'You must be on the master branch to publish' ; process.exit 1
+
+  check-npm-dependencies!
+  build-subprojects!
+  run-tests!
+  bump-version-numbers!
+  push-version-numbers!
+  publish-to-npm!
+  push-exocom-docker-image!
+  push-git-tag!
+
+
+
+published-directories =
```

Comment:
rename to `SUBPROJECTS_TO_PUBLISH` (this is a constant, so screaming snake case), and move to the top of the file, since its configuration data.